analysis/tests/lighttpd-minimal/src/main.rs

```diff
@@ -1369,6 +1369,22 @@ pub unsafe extern "C" fn lighttpd_test() {
     free(fdes /* TODO: handle cast as *mut libc::c_void */);
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn buffer_append_int(mut b: *mut buffer, mut val: intmax_t) {
+    let mut buf: [libc::c_char; 22] = [0; 22];
+    // let str: *const libc::c_char = itostr(buf.as_mut_ptr(), val);
```

Comment:
What's `itostr`?  As in, where is it defined?  It doesn't seem to be a standard C function.  Do we need to add it's declaration in the `extern` block, like there is for `buffer_append_string_len` below`.

Reply:
it's defined in `buffer.rs` in `lighttpd`, and can be ignored for now. Once https://github.com/immunant/c2rust/issues/793 is addressed we would not need to include the definition here.

Reply:
We're always going to need to include the declaration.

Reply:
What's the thing you'd like to see changed?

Reply:
Looking at the definition, https://github.com/immunant/lighttpd-rust/blob/4aec35e15309d6d66d9104f29d5d5e2b6b6f5c46/src/buffer.c#L303, it seems there's only a definition, no separate declaration, which I assumed there would be (like there are from headers).  I think it's fine to just leave things as is then.